ssl/s23_lib.c

```diff
@@ -65,6 +65,42 @@ long ssl23_default_timeout(void)
     return (300);
 }
 
+ /*
+ * ssl23_version_supported - Check that the specified `version` is supported by
+ * `SSL *` instance
+ *
+ * @s: The SSL handle for the candidate method
+ * @version: Protocol version to test against
+ *
+ * Returns 1 when supported, otherwise 0
+ */
+int ssl23_version_supported(const SSL *s, int version)
+{
+    if (s->method == SSLv23_method()) {
```

Comment:
oh gosh, you are right. Fixed. (Added `server_method` too, just for consistency).